Fix Brewlabs Adapter issue

brainstormk opened the pull request:
**NOTE**

> - If you would like to add a `volume` adapter please submit the PR [here](https://github.com/DefiLlama/adapters).
> - If you would like to add a `liquidations` adapter, please refer to [this readme document](https://github.com/DefiLlama/DefiLlama-Adapters/tree/main/liquidations) for details.

1. Once your adapter has been merged, it takes time to show on the UI. If more than 24 hours have passed, please let us know in Discord.
2. Sorry, We no longer accept fetch adapter for new projects, we prefer the tvl to computed from blockchain data, if you have trouble with creating a the adapter, please hop onto our discord, we are happy to assist you.
3. The protocol is usually listed within 24 hours of merging the PR
4. Please fill the form below  **only if the PR is for listing a new protocol** else it can be ignored/replaced with reason/details about the PR
5. **For updating listing info** It is a different repo, you can find your listing in this file: https://github.com/DefiLlama/defillama-server/blob/master/defi/src/protocols/data2.ts, you can  edit it there and put up a PR
6. Do not edit/push `package-lock.json` file as part of your changes, we use lockfileVersion 2, and most use v1 and using that messes up our CI
7. No need to go to our discord and announce that you've created a PR, we monitor all PRs and will review it asap

---
##### Name (to be shown on DefiLlama):
Brewlabs

##### Twitter Link:
https://twitter.com/TeamBrewlabs

##### List of audit links if any:
https://www.certik.com/projects/brewlabs

##### Website Link:
https://brewlabs.info/

##### Logo (High resolution, will be shown with rounded borders):
https://raw.githubusercontent.com/brewlabs-code/assets/master/blockchains/smartchain/assets/0x6aAc56305825f712Fd44599E59f2EdE51d42C3e7/logo.png

##### Current TVL:
------ TVL ------
bsc 3.83 M
polygon 28.42 k
ethereum 3.05 M

total 6.90 M

##### Chain:
Ethereum, BSC, Polygon

##### Coingecko ID (so your TVL can appear on Coingecko, leave empty if not listed): (https://api.coingecko.com/api/v3/coins/list)
{"id":"brewlabs","symbol":"brewlabs","name":"Brewlabs"}

##### Coinmarketcap ID (so your TVL can appear on Coinmarketcap, leave empty if not listed): (https://api.coinmarketcap.com/data-api/v3/map/all?listing_status=active,inactive,untracked&start=1&limit=10000)
{"id":16405,"name":"Brewlabs","symbol":"BREWLABS","slug":"brewlabs","is_active":1,"status":"active"}

##### Short Description (to be shown on DefiLlama):
Brewlabs offers individuals the opportunity to bring their cryptocurrency ideas to life through smart contract engineering, dapp/web3 development, auditing and launch. Brewlabs offers a suite of tools and DeFI products routing fee revenue back to buy back the BREWLABS token through an innovating treasury buy back model.

##### Token address and ticker if any:
Ethereum: 0xdAd33e12e61dC2f2692F2c12e6303B5Ade7277Ba, BREWLABS
BSC: 0x6aac56305825f712fd44599e59f2ede51d42c3e7, BREWLABS

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Yields

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):


##### forkedFrom (Does your project originate from another project):


##### methodology (what is being counted as tvl, how is tvl being calculated):
TVL is made up of staking, yield farming, NFT staking and indexes within the Brewlabs dAPP ecosystem.

llamatester commented:
The adapter at projects/brewlabs exports TVL: 
        
 
 
    polygon                   28.41 k
    polygon-staking           0
    ethereum                  3.05 M
    ethereum-staking          557.48 k
    bsc                       3.82 M
    bsc-staking               1.91 M
    staking                   2.47 M
    
    total                    6.90 M

llamatester commented:
The adapter at projects/brewlabs exports TVL: 
        
 
 
    polygon                   27.68 k
    bsc                       3.55 M
    ethereum                  3.04 M
    
    total                    6.62 M

g1nt0ki reviewed (changes requested):
hi @brainstormk , sorry, I dont agree with this change, we let users decide if pool2 and staking are tvl, we dont bundle them together by default: https://docs.llama.fi/list-your-project/what-to-include-as-tvl

brainstormk commented:
Hi, @g1nt0ki
I made updates based on [your update](https://github.com/DefiLlama/DefiLlama-Adapters/pull/5230) because brewlabs info was not shown in DefiLlama.

g1nt0ki commented:
> Hi, @g1nt0ki I made updates based on [your update](https://github.com/DefiLlama/DefiLlama-Adapters/pull/5230) because brewlabs info was not shown in DefiLlama.

yes, later I fixed it because tvl was wrong.

brainstormk commented:
> > Hi, @g1nt0ki I made updates based on [your update](https://github.com/DefiLlama/DefiLlama-Adapters/pull/5230) because brewlabs info was not shown in DefiLlama.
> 
> yes, later I fixed it because tvl was wrong.

But it does not work for test script. So I fixed it and make PR again.

g1nt0ki commented:
@brainstormk I am bit confused to as what you are referring to.

The only change I see in the PR is merging staking+pool2 and showing it as tvl. Users can do this on the UI if they want: 
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/99907941/56d07980-d397-4389-b80b-60a4983e0411)

brainstormk commented:
![image](https://github.com/DefiLlama/DefiLlama-Adapters/assets/109654392/86486043-6a6a-417a-bcc8-9622fbcf2243)
Our TVL is zero when pool2 and staking were not selected. So I want to fix this issue

g1nt0ki commented:
the way I see it, brew labs is providing staking infra for other projects, but in the end it is still staking, so we cant count it as tvl.

g1nt0ki commented:
closing because of inactivity
